libse/SubtitleFormats/PhoenixSubtitle.cs

```diff
@@ -71,9 +71,9 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
                 {
                     try
                     {
-                        // Decode times.
-                        paragraph.StartTime = DecodeTimeCode(match.Groups[1].Value);
-                        paragraph.EndTime = DecodeTimeCode(match.Groups[2].Value);
+                        // Read frames.
+                        paragraph.StartFrame = int.Parse(match.Groups[1].Value);
+                        paragraph.EndFrame = int.Parse(match.Groups[2].Value);
```

Comment:
Parse directly to frame otherwise we will have trouble in [Subtitle.cs:Line180.](https://github.com/SubtitleEdit/subtitleedit/blob/master/libse/Subtitle.cs#L180)
which will try calculate **TimeCode** using **Frame**.